atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitions.java

```diff
@@ -160,11 +162,11 @@ static CfDef getStandardCfDef(String keyspace, String internalTableName) {
     @SuppressWarnings("CyclomaticComplexity")
     static boolean isMatchingCf(CfDef clientSide, CfDef clusterSide) {
         String tableName = clientSide.name;
-        if (!Objects.equal(clientSide.compaction_strategy_options, clusterSide.compaction_strategy_options)) {
+        if (clientSide.compaction_strategy != clusterSide.compaction_strategy) {
```

Comment:
does `CfDef` not implement `equals` correctly? How does this work?

Reply:
compaction_strategy_options was a HashMap that could be null, so you can't just clientHashMap.equals(otherHashMap) without it possible NPEing.
now I'm only checking the string that either says "LCS" or "STCS",
since we weren't setting the options map correctly before / not setting it now

Reply:
oh I see what you're saying also.
Yes, this whole class is because CfDef doesn't implement equals sanely.

Basically the normal upstream CfDef.equals fails on defaults being specified vs unspecified, which is very annoying.